src/reducers/notebook-reducer.js

```diff
@@ -89,11 +92,6 @@ const notebookReducer = (state = newNotebook(), action) => {
         }),
       }, { title: state.title === undefined ? 'new notebook' : state.title })
```

Comment:
after your other nice refactoring, do we still need `{ title: state.title === undefined ? 'new notebook' : state.title }`? Can the title ever be undefined at this point? If not, we can remove all the checks for that, and we can even update the state schema in state-prototypes.js, the fixme around line 80:
`title: {}, // FIXME change to string ONLY with default '' or 'untitled'`